src/Centreon/Application/Controller/PlatformController.php

```diff
@@ -212,6 +212,8 @@ public function updatePlatform(Request $request): View
                     case 'peerValidation':
                         $platformInformationUpdate->setApiPeerValidation($platformValue);
                         break;
+                    case 'platformName':
+                        $platformInformationUpdate->setPlatformName($platformValue);
```

Comment:
Even if it does not change behaviour, this suggestion is to stay consistent with other cases
```suggestion
                        $platformInformationUpdate->setPlatformName($platformValue);
                        break;
```